asc2018/vdr/electrical/WaterlooMidnightSun_ASC2018VDR_BatteryProtectionReport.tex

```diff
@@ -0,0 +1,192 @@
+\urlstyle{same}
+
+\newcommand\theteamname{Midnight Sun Solar Car Team} % should not change normally
+\newcommand\theuniversityname{University of Waterloo} % should also not change normally
+\newcommand\theteamwebsite{www.uwmidsun.com} % should also not normally change
+\newcommand\theteamphone{(519) 888-4567 x32978} % should also not normally change
+
+\newcommand\thetitle{Battery Protection Tech Report} % <--------------- add the title
+\newcommand\thesubtitle{Electrical} % <--------------- add a subtitle or leave the field empty
+\newcommand\theauthor{Minghao Ji} % <--------------- add an author with contact info or comment this line
+\newcommand\theauthorcontact{minghao.ji@uwmidsun.com} % <--------------- add author's email or leave the field empty
+\newcommand\thedate{\today} % <--------------- update the date, use this format
+
+\pagestyle{fancy}
+\renewcommand{\headrulewidth}{0pt}
+\lhead{\thetitle}
+\rhead{\theteamname}
+
+\begin{document}
+
+% Title Page
+\begin{titlepage}
+\large
+\vspace*{2cm}
+\centering
+\includegraphics[width=.25\textwidth]{./figures/midnightSunLogoCircle.png} \\
+\vspace{1.5cm}
+{\LARGE \theteamname} \\
+\theuniversityname \\
+\vspace{2.2cm}
+{\LARGE MSXII} \\
+\vspace{0.4cm}
+{\huge\bfseries \thetitle} \\
+\vspace{0.2cm}
+{\LARGE \thesubtitle} \\
+\vspace{2.2cm}
+\ifdefined \theauthor
+\par Prepared by: \\
+\theauthor \\
+\theauthorcontact \\
+\fi
+\thedate \\
+\vfill
+\theteamwebsite \\
+\theteamphone
+\end{titlepage}
+
+% Main Matter
+\tableofcontents
+\newpage
+%\listoffigures % <-------------- uncomment for list of figures
+%\listoftables % <-------------- uncomment for list of tables
+
+\section{Overview}
+
+\subsection{5.2.E.1: Battery Approval Form}
+
+See attached form
+
+\includepdf[pages={-}]{forms/battery_approval_form.pdf}
+
+\subsection{5.2.E.2: Battery Pack Configuration}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Cells per module}    & 36   \\ \hline
+\textbf{Modules per string}  & 36   \\ \hline
+\textbf{Strings in parallel} & 1    \\ \hline
+\textbf{Total cell count}    & 1296 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.3: Over temperature set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over temperature set point (charge) ($^{\circ}$C)}    & 45 \\ \hline
+\textbf{Over temperature set point (discharge) ($^{\circ}$C)} & 60 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.4: Under voltage set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Under voltage set point (V)}    & 2.5 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.5: Over voltage set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over voltage set point (V)}    & 4.2 \\ \hline
+\end{tabular}
+\end{table}
+
+\subsection{5.2.E.6: Over current set points}
+
+\begin{table}[!htbp]
+\begin{tabular}{|l|l|}
+\hline
+\textbf{Over current set point (A)}    & 110 \\ \hline
```

Comment:
Should we bump this higher?

Copying my comment from the general VDR PR:

> We chose `110 A` because `(7.5 kW / 160 V) * 2 = 46.87500 A * 2`, but that's using `4.2 V`, which is the maximum of the 18650s, plus like `10 A` of headroom.
> 
> Shouldn't we at least be using the nominal (`3.635 V`), in which case we have `(7.5 kW / 130.86 V) * 2 = 57.3131591 * 2 = 114.626318 A`, which is already above `110 A`?

Reply:
We should set this to at least 140 A (3 V per cell), but probably can set it up to 170 A (2.5 V) since the fuse should be able to handle that for 30 s +.

Reply:
Copying from electrical VDR:

> We can increase the current limit, but 110A is safe based on wire ratings and fuse derating. If we believe the wire gauge charts, our wire can support up to 215A. Per regulations, our fuse needs to be rated for less than 75% of our wire's current capacity. That means our fuse needs to be less than 160A, so we need to use a 150A fuse. If we derate by 25%, we get 112A.

What fuse are you referring to @taipingli? The fuse we currently have is not able to handle 170A at all.

I don't think we need to be designing for hitting the maximum motor output. Based on the WaveSculptor datasheets, it seems like the peak power scenario is when we attempt to accelerate at high speeds, as power is a function of torque * speed. (See [this chart](http://www.tritium.com.au/wp-content/uploads/2017/06/TRI50.019_Motor_Impedance_v1.pdf)) Based on WSC and ASC results, it seems like ~65km/h is a decent target to aim for. Also good to note that our motor controllers are only rated up to 50A continuous and 100A peak, so I assume we shouldn't be aiming to be the peak zone for too long.

Also, the voltage drop we'll see from loading the pack at 110A will probably be around ~250mV per cell based on the [discharge charts](http://lygte-info.dk/review/batteries2012/LG%2018650%20MJ1%203500mAh%20%28Green%29%20UK.html) I've seen, so that's something we should keep in mind.